pkg/block/blocktest/multipart_suite.go

```diff
@@ -211,15 +211,14 @@ func verifyListInvalid(t *testing.T, ctx context.Context, adapter block.Adapter,
 	if expectOperationNotSupported(adapter.BlockstoreType()) {
 		require.ErrorIs(t, err, block.ErrOperationNotSupported)
 	} else {
-		require.NotNil(t, err)
+		require.NotNil(t, err, "ListParts should have failed with invalid uploadID")
```

Comment:
Not introduced here, but this _really_ needs to verify the error.

Reply:
added a comment in the code that we check for any error until all adapters will use the same error or wrapped error.